src/jit/emitxarch.cpp

```diff
@@ -5062,6 +5065,31 @@ void emitter::emitIns_AR_R(instruction ins, emitAttr attr, regNumber ireg, regNu
     emitAdjustStackDepthPushPop(ins);
 }
 
+#ifndef LEGACY_BACKEND
+void emitter::emitIns_AR_R_I(instruction ins, emitAttr attr, regNumber base, int disp, regNumber ireg, int ival)
+{
+    assert(ins == INS_vextracti128 || ins == INS_vextractf128);
+    assert(base != REG_NA);
+    assert(ireg != REG_NA);
+    UNATIVE_OFFSET sz;
+    instrDesc*     id = emitNewInstrAmdCns(attr, disp, ival);
+
+    id->idIns(ins);
+    id->idInsFmt(IF_AWR_RRD_CNS);
+    id->idAddr()->iiaAddrMode.amBaseReg = base;
+    id->idAddr()->iiaAddrMode.amIndxReg = REG_NA;
+    id->idReg1(ireg);
+
+    assert(emitGetInsAmdAny(id) == disp); // make sure "disp" is stored properly
+
+    sz = 6;
```

Comment:
Comment as to why 6? I believe we have these elsewhere...

Reply:
It would also be good to understand why the size calculation functions can't be used. I think those should be preferred, if possible

Reply:
Only `vextracti/f128` needs this function, so the code size is known and we do not need to calculate it. Meanwhile, the current code-size estimation is not accurate for `vextracti/f128`.

Will add a comment.